cmd/blobstream/keys/evm/evm.go

```diff
@@ -379,6 +426,87 @@ func ImportECDSA(serviceName string) *cobra.Command {
 	return keysConfigFlags(&cmd, serviceName)
 }
 
+func ImportMnemonic(serviceName string) *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "mnemonic",
+		Args:  cobra.ExactArgs(0),
+		Short: "import an EVM address from a 24 words BIP39 mnemonic phrase",
```

Comment:
if the key is being used to sign something and not just an address, then I think this terminology makes more sense 
```suggestion
		Short: "import a EVM private key from a 24 words BIP39 mnemonic phrase",
```